app/pages/projects/project-filtering-interface.jsx

```diff
@@ -68,10 +71,11 @@ class ProjectFilteringInterface extends Component {
             const meta = projects[0].getMeta();
             pages = meta.page_count;
             projectCount = meta.count;
+            pageSize = meta.page_size;
           }
-          this.setState({ projects, pages, projectCount });
+          this.setState({ projects, pages, projectCount, pageSize });
         } else {
-          this.setState({ projects: [], pages: 0, projectCount: 0 });
+          this.setState({ projects: [], pages: 0, projectCount: 0, pageSize: 20 });
```

Comment:
the server defaults to 20 and it used to be 20 (`pageStart = ((this.props.page - 1) * 20) + 1;`) and i figured it'd have something for the calcs below (though probably not needed).